js/lib/cli.js

```diff
@@ -326,6 +349,12 @@ function processInputSync(filepath) {
 
 function makePretty(code, config, outfile, callback) {
     try {
+        if (config.editorConfig) {
```

Comment:
@ethanluoyc - I see what you're saying.  The alternative is to say that editorconfig values superceed all other config values.  That could be acceptable - the user is specifying "Use EditorConfig". 

We still need tests that show and enforce that behavior.